.changes/config-refactor.md

```diff
@@ -0,0 +1,21 @@
+---
+"tauri-utils": minor
+"tauri-api": minor
+"tauri": minor
+---
+
+The Tauri files are now read on the app space instead of the `tauri` create.
+Also, the `AppBuilder` `build` function now returns a Result.
+
```

Comment:
Please don't put usage in the changelog. This should be put in a README / in the inline documentation / on the site.

Reply:
oops :D